### PR TITLE
direct content lists publishing + fixes

### PR DESCRIPTION
**Pull request by tomaszrondio**
collapsible pane in content lists
content lists scrollable dropdown
search input debounce fix
content lists preview content image fix

### CODE REVIEW

File: client/controllers/WebPublisherOutputController.js [hunk 1]
```diff
@@ -359,6 +359,8 @@ export function WebPublisherOutputController($scope, $sce, modal, publisher, pub
                 });
             });
 
+            console.log(updatedKeys, destinations);
```
Comment (ahilles107) [inline client/controllers/WebPublisherOutputController.js]: please remove it

File: client/extensions/targetedPublishing/Tenant.js [hunk 2]
```diff
@@ -26,14 +28,15 @@ export default class Tenant extends Component {
...
+        this.getContentLists();
     }
 
     componentWillReceiveProps(nextProps) {
         if (nextProps.rule) {
             const rule = nextProps.rule;
             this.setState({
-                rule: rule,
-                newRule: rule
+                rule: JSON.parse(JSON.stringify(rule)),
```
Comment (ahilles107) [inline client/extensions/targetedPublishing/Tenant.js]: why you hack it like that?

File: client/extensions/targetedPublishing/NewDestination.js [hunk 3]
```diff
@@ -91,6 +104,25 @@ export default class NewDestination extends Component {
 
     }
 
+    saveContentListsHandler = (data) => {
+        let destination = {...this.state.destination};
+        destination.contentLists = data;
+        console.log(data);
```
Comment (ahilles107) [inline client/extensions/targetedPublishing/NewDestination.js]: remove log

File: client/extensions/targetedPublishing/Tenant.js (hunk 2 above)
Comment (tomaszrondio) [reply client/extensions/targetedPublishing/Tenant.js]: that is one of two ways to deeply copy an object

**Pull request merged**